tests/lit-tests/2280.ispc

```diff
@@ -0,0 +1,33 @@
+// RUN:     %{ispc} %s --target=host --nostdlib --emit-llvm-text -E                       -o -
```

Comment:
Wasn't sure how to check for null output (empty stdin && no generated files). Checks for compiler status code.

Reply:
Good point! Default behavior for C compilers (clang, gcc and others) is to dump the output to stdout, i.e. assume `-o -` in this case. We should do the same.

Reply:
As for checking that no "object file output produced, I think it's ok not to check that.